src/status_im/contexts/wallet/swap/events.cljs

```diff
@@ -44,12 +45,15 @@
          network'               (or network
                                     (swap-utils/select-network asset-to-pay))
          start-point            (if open-new-screen? :action-menu :swap-button)]
-     {:db (-> db
-              (assoc-in [:wallet :ui :swap :asset-to-pay] asset-to-pay)
-              (assoc-in [:wallet :ui :swap :asset-to-receive] received-asset)
-              (assoc-in [:wallet :ui :swap :network] network')
-              (assoc-in [:wallet :ui :swap :launch-screen] view-id)
-              (assoc-in [:wallet :ui :swap :start-point] start-point))
+     {:db
+      (update-in db
+                 db/swap
+                 assoc
+                 :asset-to-pay     asset-to-pay
+                 :asset-to-receive received-asset
+                 :network          network'
+                 :launch-screen    view-id
+                 :start-point      start-point)
```

Comment:
I'd prefer 
```clojure
{:db (update-in db db/swap assoc 
    ;; ... pairs to assoc
 }
```

But I think our tool to format forces us to do this

Reply:
yeah, I tried, it is autoformatted(